### PR TITLE
Authentication  and get meetings

### PR DESCRIPTION
**Pull request by ArielRojas**

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-	"url" : "https://172.20.208.194:4040",
+	"url" : "https://172.20.208.195:4040",
 	"timeOut" : 4000,
 	"userAccountJson" : {
-		"username": "test",
```
Comment (AlejandroJeanCarlo) [inline config/config.json]: do not forget stay this please

File: lib/meetingsAPI.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ * @roomId {string} is a parameters that contains the room id
+ * @callback contains the function instance  
+ */
+var getMeetings = function(serviceId,roomId,callback){
+	var meetingsByIdRoomEndPoint = meetingsByIdEndPoint.replace('{:serviceId}', serviceId);
+	var meetingsByIdRoomEndPoint = meetingsByIdRoomEndPoint.replace('{:roomId}', roomId);
+	console.log('GET ' + meetingsByIdRoomEndPoint);
+	request
+		.get(url + meetingsByIdRoomEndPoint)
+	.end(function(err, res){
```
Comment (AlejandroJeanCarlo) [inline lib/meetingsAPI.js]: please tab this

File: lib/meetingsAPI.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+exports.getService = getService;
+
+/**
+ * This method helps to get the rooms, the paramenters that needs are:
+ * @callback contains the function instance
+ */
+var getRooms = function(callback){
+	request
+		.get(url + roomByIdEndPoint)
+	.end(function(err, res){
```
Comment (AlejandroJeanCarlo) [inline lib/meetingsAPI.js]: same

File: lib/tokenAPI.js [hunk 4]
```diff
@@ -1,8 +1,6 @@
-// Token API
-//Jean Carlo Rodriguez
 var request = require('superagent');
-var endPoint = require('../config/endPoints.json');
 var config = require('../config/config.json');
+var endPoint = require(config.path.endPoints);
```
Comment (AlejandroJeanCarlo) [inline lib/tokenAPI.js]: this would broke the code please maintain '../config/endPoints.json'

File: config/config.json (hunk 1 above)
Comment (ArielRojas) [inline config/config.json]: Ok

File: lib/meetingsAPI.js (hunk 2 above)
Comment (ArielRojas) [inline lib/meetingsAPI.js]: Ok

File: lib/tokenAPI.js (hunk 4 above)
Comment (ArielRojas) [inline lib/tokenAPI.js]: Ok

**Comment by AlejandroJeanCarlo** [comment]
remember always pull the code from the main repository before push your code

File: config/config.json [hunk 5]
```diff
@@ -10,10 +10,11 @@
 		"Ok": 200
 	},
 	"path":{
-		"tokenAPI": "../../lib/tokenAPI",
+		"tokenAPI" : "../../lib/tokenAPI",
 		"resourcesAPI" : "../../lib/resourcesAPI",
-		"endPoints" : "../../config/endPoints.json",
```
Comment (AlejandroJeanCarlo) [inline config/config.json]: do not delete this please

File: lib/meetingsAPI.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
+//get meetings
+//Author Ariel Wagner Rojas
+var request = require('superagent');
+require('superagent-proxy')(request);
```
Comment (AlejandroJeanCarlo) [inline lib/meetingsAPI.js]: delete this

File: tests/Smoke/authenticationLocal.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
+//Smoke testing - POST Authentication
+//Author Ariel Wagner Rojas
+var request = require('superagent');
+require('superagent-proxy')(request);
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/authenticationLocal.js]: delete this too please

File: tests/Smoke/meetings.js [hunk 8]
```diff
@@ -0,0 +1,56 @@
+//Smoke testing - GET Meetings
+//Author Ariel Wagner Rojas
+var request = require('superagent');
+require('superagent-proxy')(request);
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/meetings.js]: same deleted please

File: tests/Smoke/authenticationLocal.js [hunk 9]
```diff
@@ -0,0 +1,28 @@
+//Smoke testing - POST Authentication
+//Author Ariel Wagner Rojas
+var request = require('superagent');
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/authenticationLocal.js]: delete this too please

File: tests/Smoke/meetings.js [hunk 10]
```diff
@@ -0,0 +1,56 @@
+//Smoke testing - GET Meetings
+//Author Ariel Wagner Rojas
+var request = require('superagent');
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/meetings.js]: delete this too please

File: lib/meetingsAPI.js (hunk 6 above)
Comment (ArielRojas) [inline lib/meetingsAPI.js]: Ok

**Pull request merged**